blog/pymc3_vs_pymc_v4.ipynb

```diff
@@ -1,7 +1,6 @@
 {
```

Comment:
<p>The image is using the html syntax instead of markdown and doesn't appear in the <a href="https://pymcio--14.org.readthedocs.build/blog/pymc3_vs_pymc_v4.html" target="_blank">website</a>. html is treated as raw content but doing that breaks the image path, markdown image syntax is understood by sphinx that will add the image to the website and adapt the path in the generated html image as necessary.</p><p> </p><p>I also think this should be a figure with the current alt text as caption as the current alt text is useful and interesting to everybody, not only people who use screen readers and the credit part doesn't really serve as alt text as it doesn't help people using a screen reader get an idea of what the image is about.</p><p> </p><p> </p><p> </p><p>How about making the "what about pymc4?" section an aside at the margin? The theme provides two options, <a href="https://sphinx-book-theme.readthedocs.io/en/stable/content-blocks.html#margin-content" target="_blank">margin content</a> or <a href="https://sphinx-book-theme.readthedocs.io/en/stable/content-blocks.html#content-sidebars" target="_blank">sidebar content</a> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/pymc-devs/pymc.io/pull/14/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>